src/emulator/functionsEmulator.ts

```diff
@@ -819,7 +862,7 @@ export class FunctionsEmulator implements EmulatorInstance {
     logger.debug(`addPubsubTrigger`, JSON.stringify({ eventTrigger }));
 
     // "resource":\"projects/{PROJECT_ID}/topics/{TOPIC_ID}";
-    const resource = eventTrigger.resource;
+    const resource = eventTrigger.resource!;
```

Comment:
is this ! necessary here?

Reply:
We make def.resource `string | undefined` now